code/modules/mob/living/carbon/xenomorph/castes/widow/abilities_widow.dm

```diff
@@ -269,11 +275,14 @@
 	X.mouse_opacity = initial(X.mouse_opacity)
 	X.density = TRUE
 	X.allow_pass_flags &= ~PASSABLE
-	REMOVE_TRAIT(X, TRAIT_IMMOBILE, WIDOW_ABILITY_TRAIT)
-	REMOVE_TRAIT(X, TRAIT_BURROWED, WIDOW_ABILITY_TRAIT)
-	REMOVE_TRAIT(X, TRAIT_HANDS_BLOCKED, WIDOW_ABILITY_TRAIT)
+	REMOVE_TRAIT(X, TRAIT_IMMOBILE, ability_name)
```

Comment:
What the fuck? No, the source should always be a define.

Reply:
These were me trying to figure out why the spiders were able to move once the widow gets up, burrow and resting, and waking out of critical are the main sources of issues, and I thought it was because the source was re-used in several places, and the widow has several ways of removing and re-adding this trait and source. There was a different spot that also used this same source method, so I used it here, I'll revert the trait changes